css/at-rules/media.json

```diff
@@ -1383,16 +1383,16 @@
                 "version_added": true
               },
               "opera_android": {
-                "version_added": null
+                "version_added": true
               },
               "safari": {
-                "version_added": null
+                "version_added": "6.1"
               },
               "safari_ios": {
-                "version_added": null
+                "version_added": "7"
               },
               "samsunginternet_android": {
-                "version_added": null
+                "version_added": "1.5"
               },
               "webview_android": {
                 "version_added": "37"
```

Comment:
This should be `true`. I think that's what we've settled on for features that landed before WebView became a Chromium fork.